taichi/backends/cuda/cuda_driver.cpp

```diff
@@ -23,6 +23,7 @@ CUDADriver::CUDADriver() {
   loader = std::make_unique<DynamicLoader>("libcuda.so");
 #else
   loader = std::make_unique<DynamicLoader>("nvcuda.dll");
+  // TODO(yuanming-hu): so what about OSX?
```

Comment:
We don't plan to support OS X but I'll make it clearer here. Thanks for pointing this out.